lib/ansible/plugins/action/__init__.py

```diff
@@ -287,11 +287,14 @@ def _make_tmp_path(self, remote_user=None):
         if rc == '/':
             raise AnsibleError('failed to resolve remote temporary directory from %s: `%s` returned empty string' % (basefile, cmd))
 
+        self._connection._shell.tempdir = rc
+
+        if not use_system_tmp:
+            self._connection.shell.env.update('ANSIBLE_REMOTE_TEMP': self.tempdir)
```

Comment:
Syntax error here which is is causing a lot of the failures

Reply:
fixed, thnx